docs/clients/brushless_drive.rst

```diff
@@ -105,72 +105,116 @@ Message Table
+| 41     | derate                        | get            | int32     | PU fix16 | Amount of derating. No derate = 65536, full derate = 0                                                                               |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 44     | emf                           | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 45     | volts_at_max_amps             | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 46     | slew_volts_per_second         | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 47     | slew_enable                   | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 48     | motoring_supply_current_limit | get, set, save | float     | fix16_t  |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 49     | regen_supply_current_limit    | get, set, save | float     | fix16_t  |                                                                                                                                      |      |
```

Comment:
Description: "On modules that support dynamic supply current limiting, this entry determines the current limit for the current back to the supply when the module is regenerating, i.e. the back-EMF is greater than the drive voltage, as opposed to when it is motoring. Note this is the current from the supply, so it would be the current from your battery or power supply."

Units are A